src/util/std_expr.h

```diff
@@ -629,8 +589,7 @@ class unary_predicate_exprt:public unary_exprt
   using exprt::op2; // hide
 };
 
-/*! \brief sign of an expression
-*/
+/// \brief sign of an expression
```

Comment:
This needs some explanation. When is this predicate true/false?

Reply:
Added description

Reply:
Nit: Upper case S